spec/features/repair_request_spec.rb

```diff
@@ -300,13 +300,14 @@ def stub_property_temp_annex
 
       expect(page).to have_content "CC"
       expect(page).to have_link("Launch Keyfax", href: "https://www.keyfax.com")
+      expect(find_by_id('hackney_repair_request_description')).to have_content 'CC'
 
       stub_keyfax_get_results_response
 
       visit new_property_repair_request_path('00000666', status: "1", guid: '123456789')
 
       select "N - Normal"
-      fill_in "Problem description", with: "It's broken"
+      fill_in "Problem description", with: "CC It's broken"
```

Comment:
We should check if this comes pre-filled instead of filling it, both before and after Keyfax!

Reply:
👍 we also select normal, but shouldn't we be checking that it's already normal?

Reply:
If we do prefill with info from Keyfax, then yes, we should.